runway/__version__.py

```diff
@@ -1 +1 @@
-__version__ = '0.0.67'
```

Comment:
We should update the CHANGELOG to describe these changes. The bullet points you put in the PR would suffice :)